frontend/package.json

```
@@ -9,7 +9,9 @@
     "preview": "nuxt preview",
     "postinstall": "nuxt prepare",
     "lint": "eslint .",
-    "lint:fix": "eslint . --fix"
+    "lint:fix": "eslint . --fix",
+    "i18n-extract": "vue-i18n-extract report --vueFiles './{components,pages,layouts,composables}/**/*.?(ts|vue)' --languageFiles './locales/*.?(json|yml|yaml|js)' --add",
```

My poor Windows machine can't deal with this syntax. I had to adapt it to:

```suggestion
    "i18n-extract": "vue-i18n-extract report --vueFiles \"./{components,pages,layouts,composables}/**/*.{ts,vue}\" --languageFiles \"./locales/**/*.{json,yml,yaml,js}\" --add",
```

Does that work on Linux as well?

---

`docker compose run vue-dev npm run i18n-extract` works on everything :)

I would prefer that we avoid running these scripts outside of the container. That way we can prevent differences in package versions and OSes

---

That being said your suggested command does work in the container as well (AFAICT) so I guess that for now you can have your cake and eat it

---

I like cake. 🍰

In all seriousness, I like being able to run a dev build outside of Docker as well (just to remove one more layer of abstraction in case of debugging) so I appreciate being able to run this command on my Windows machine as well 😊

---

OK, so we can keep it as is? :)

---

I think @miggol meant that my suggestion works both on Linux and Windows, so it we would use that, that would be great 😇